commit 18fcc367d28da709cfdafef35d14861ea721def4
Merge: 1d06b10 19ce90f
Author: daid <daid303@gmail.com>
Date:   Sat Feb 2 03:23:43 2013 -0800

    Merge pull request #371 from Banjer/Melzi_LED_PIN
    
    Chenged LED_PIN for Melzi boards, added comment about broken versions of...

diff --git a/Marlin/Marlin_main.cpp b/Marlin/Marlin_main.cpp
index 473d205..39efdf4 100644
--- a/Marlin/Marlin_main.cpp
+++ b/Marlin/Marlin_main.cpp
@@ -88,7 +88,7 @@
 // M29  - Stop SD write
 // M30  - Delete file from SD (M30 filename.g)
 // M31  - Output time since last M109 or SD card start to serial
-// M42  - Change pin status via gcode
+// M42  - Change pin status via gcode Use M42 Px Sy to set pin x to value y, when omitting Px the onboard led will be used.
 // M80  - Turn on Power Supply
 // M81  - Turn off Power Supply
 // M82  - Set E codes absolute (default)

